src/validator.js

```diff
@@ -0,0 +1,12 @@
+class Validator {
+
+  static isString(str) {
+    return typeof str === 'string' || str instanceof String;
```

Comment:
@dwalintukan js newbie question, is the check duplicated?

Reply:
JavaScript automatically converts primitives string value to String objects, so most of time ppl use these two criteria to check